source/_components/media_player.anthemav.markdown

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: "Anthem A/V Receivers"
+description: "Instructions how to integrate Anthem A/V Receivers into Home Assistant."
+date: 2016-12-31 23:11
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: anthemav.png
+ha_category: Media Player
+ha_iot_class: "Local Push"
+ha_release: 0.36
+---
+
+Both [Anthem]'s current and last generation of A/V Receivers and and Processors
+support IP-based, network control.  This Home Assistant platform adds proper
+"local push" support for any of these receivers on your network.
+
+## {% linkable_title Supported Models %}
+
+* MRX 520, MRX 720, MRX 1120, and AVM 60
+* MRX 310, MRX 510, MRX 710
+
+Support is provided through the Python [anthemav] module.  Older, RS-232
+serial-based units like the D2v use a different protocol entirely and are not
+currently supported.
+
+[Anthem]:	http://www.anthemav.com/
+[anthemav]: https://github.com/nugget/python-anthemav
+
+
+## {% linkable_title Configuration %}
+
+```yaml
+# Example configuration.yaml entry
+media_player:
+  - platform: anthemav
+    host: IP_ADDRESS
+    port: 14999
+```
+
+### Supported Options
```

Comment:
Can you update the list of options to match the format of other pages? [One example here](https://github.com/home-assistant/home-assistant.github.io/blob/current/source/_components/media_player.kodi.markdown).